.github/workflows/main.yml

```diff
@@ -61,6 +62,7 @@ jobs:
     - uses: actions/checkout@v5
     - run: rustup update --no-self-update stable && rustup default stable
     - run: rustup target add wasm32-unknown-unknown
+    - run: rustup component add clippy rustfmt
```

Comment:
Might be a good idea to merge this into `stable` installation above, like you did for nightly, so that it does everything in one go.